src/test/resources/configurations/test-conf-missing.json

```diff
@@ -0,0 +1,27 @@
+{
+  "jwt.public": "executables/collector/src/test/resources/public.pem",
+  "http.host": "localhost",
+  "http.endpoint": "/api/v3",
+  "http.port": 8080,
+  "mongo.db": {
+    "db_name": "cyface",
+    "connection_string": "mongodb+srv://user:password@mongodb.net/?retryWrites=true&w=majority",
+    "data_source_name": "cyface"
+  },
+  "admin.user": "admin",
+  "admin.password": "secret",
+  "salt": "cyface_salt",
+  "jwt.expiration": 60,
+  "upload.expiration": 60,
```

Comment:
You probably meant `1000 * 60 * 1` ?

Reply:
No. I just required a value here. These files are just used to test parsing, not to really start a server.

Reply:
I would just be confused by "60" for milliseconds ;-) 1 or 1_000 or 60_000 is better, thanks for changing that :)